### ISSUE TITLE
Bot is making "empty" posts

### ISSUE THREAD

**Issue by Spootacular** [Spootacular/Mnemosyne-2.0#9]
@chuggafan Noticing this [here](https://www.reddit.com/r/KotakuInAction/comments/4nxtp0/stop_killing_off_characters_of_color_to_give/) and [here](https://www.reddit.com/r/KotakuInAction/comments/4nxryu/eli5_why_does_it_take_a_mass_murder_of/) for example. I've got some stuff all up in my face so I won't really be able to look into it for a few hours

**Comment by chuggafan** [comment]
Yhea, i noticed, i'll leave it like that for a bit, but will fix now

**Comment by Spootacular** [comment]
This "should" be fixed now but I'll leave the issue open for a bit as I haven't tested. I get errors on build for two missing files ArchiveLinkDatabase.mdf and ArchiveLinkDatabase_log.ldf being ignored by git

**Comment by chuggafan** [comment]
Ah, i'll remove the two then and commit and pull
